src/Algebra/Apartness/Structures.agda

```diff
@@ -34,6 +34,7 @@ record IsHeytingCommutativeRing : Set (c ⊔ ℓ₁ ⊔ ℓ₂) where
 
   open IsCommutativeRing isCommutativeRing public
   open IsApartnessRelation isApartnessRelation public
+    renaming (sym to #-sym)
```

Comment:
I think we should take this opportunity to rename all three fields with a `#` to be consistent?

Reply:
Do you mean in-place in this `import`, or actually making the `breaking` change back in the `record` definition in `Relation.Binary.Structures.IsApartnessRelation`?

Reply:
Good question. In this import!